python/cudf/io/csv.py

```diff
@@ -37,6 +41,8 @@ def read_csv(filepath, lineterminator='\n',
         Determines whether to use whitespace as delimiter.
     lineterminator : char, default '\\n'
         Character to indicate end of line.
+    windowslinetermination : bool, default False
+        Indicates whether the file is a windows file
```

Comment:
A user shouldn't have to specify a line terminator as `\n` vs `\r\n`, they should both work as the default.

Reply:
True; in that case, I don't know why we need this parameter then?

Reply:
The user technically doesn't explicitly provide the line terminator for a windows file, just passes the windowslinetermination boolean as True, this is updated in the csv struct for using correct line terminator.

Reply:
That is actually correct, I went through the code, looks like it would be something required if the user directly uses read_csv from C, since there are no default values currently and lineterminator cannot be passed \r\n. Python can do away with it.